src/main/java/com/coveo/pushapiclient/Sandbox.java

```diff
@@ -0,0 +1,88 @@
+package com.coveo.pushapiclient;
+
+import java.util.HashMap;
+import java.util.List;
+
+public class Sandbox {
+
+    /**
+     * Upload a document batch in push mode -> using the /files endpoint
+     */
+    public static void uploadBatch() {
+        CatalogSource source = new CatalogSource("my_api_key", "my_org_id");
+
+        // Prepare the push in "Update" mode. In this mode, the push Service will create
+        // S3 file containers with appropriate batches sizes.
+        DocumentUpdateService service = source.startDocumentUpdate("my_source_id");
+
+        // Create a stream for pushing documents
+
+        // Prepare a list of documents to add or update from an imaginary method
+        // The SDK handles the batching. Simply feed documents into the service
+        for (DocumentBuilder document : prepareDocuments()) {
+            service.addOrUpdateDocument(document);
+        }
+
+        // Prepare a list of documents to partially update from an imaginary method
+        // Same method can be used for both full and partial updates
+        for (PartialUpdateDocument document : preparePartialUpdateDocuments()) {
+            service.addOrUpdateDocument(document);
```

Comment:
Would `partialUpdate` be a clearer method name? Considering the [possible operators](https://docs.coveo.com/en/l62e0540/coveo-for-commerce/how-to-update-your-catalog#partial-document-update-operators), it doesn't feel "symmetrical" to mention `add` and not `remove`. If `remove` is part of `update`, then why mention `add`,

Reply:
ohhh. that's a very good point. Let's do that.